src/Microsoft.TemplateEngine.Edge/Settings/SettingsLoader.cs

```diff
@@ -97,37 +95,14 @@ private IMountPointManager MountPointManager
             }
         }
 
-        private SettingsStore UserSettings
-        {
-            get
-            {
-                var local = _userSettings;
-                if (local != null)
-                {
-                    return local;
-                }
-
-                lock (_settingsLock)
-                {
-                    if (_userSettings == null)
-                    {
-                        EnsureLoaded();
-                    }
-                    //EnsureLoaded sets _userSettings
-                    return _userSettings!;
-                }
-            }
-        }
-
         public void Save()
```

Comment:
should be marked obsolete

Reply:
Do we need to mark this on implementation as well? I would assume interface is sufficient... Also this class is being deleted soon...